tests/kas.spec.ts

```diff
@@ -120,3 +120,9 @@ test('test fail to create Kafka instance with the same name', async ({ page }) =
 
   await deleteKafkaInstance(page, testInstanceName);
 });
+
+// test_4kafka.py test_kafka_404_not_found_global
+test('404 not found Kas UI', async ({ page }) => {
```

Comment:
This test doesn't need any special preparation (such as cleaning Kafka instances).
The best file to place it would be `tests/auth.ts`, and maybe rename `auth` to something more generic at this point.

Reply:
@andreaTP please review on https://github.com/redhat-developer/consoledot-e2e/pull/31 as I opened new PR because of name switching